### PR TITLE
Ms grafana oauth

### PR DESCRIPTION
**Pull request by smark88**

### CODE REVIEW

File: ops/k8s/configs/oauth.yaml.ctmpl [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+{{ with $suffix := env "SUFFIX" }}
+{{ with $project := env "GOOGLE_CLOUD_PROJECT" }}
+{{ with secret (printf "secret/dsde/datarepo/%s/clientid-secret.json" $environment ) }}
+
+apiVersion: v1
+kind: Secret
+metadata:
+  name: google-clientid-secret
+data:
+  clientid_secret: {{ .Data.oauthClientSecret | base64Encode }}
```
Comment (thathert) [inline ops/k8s/configs/oauth.yaml.ctmpl:11]: Let's make all of these camelCase

File: ops/k8s/deployments/grafana.yaml [hunk 2]
```diff
@@ -58,12 +58,54 @@ spec:
...
+          valueFrom:
+            secretKeyRef:
+              name: google-clientid-secret
+              key: adminpassword
         - name: GF_INSTALL_PLUGINS
           value: "grafana-kubernetes-app,grafana-worldmap-panel,doitintl-bigquery-datasource"
+        - name: GF_AUTH_GOOGLE_ENABLED
+          value: "true"
+        - name: GF_AUTH_GOOGLE_ALLOW_SIGN_UP
+          value: "true"
```
Comment (thathert) [inline ops/k8s/deployments/grafana.yaml:75]: What does this mean? I think we only want to allow people with Broad/Verily accounts in. Does this line up with what you're thinking?

**Review by thathert — CHANGES_REQUESTED** [review]
Styling and a question

File: ops/k8s/deployments/grafana.yaml (hunk 2 above)
Comment (smark88) [reply ops/k8s/deployments/grafana.yaml:75]: It is only Broad Verily allowed, it sends out the oauth request and then stores the user locally

File: ops/k8s/configs/oauth.yaml.ctmpl (hunk 1 above)
Comment (smark88) [reply ops/k8s/configs/oauth.yaml.ctmpl:11]: are you referring to `clientid_secret`

File: ops/k8s/deployments/grafana.yaml (hunk 2 above)
Comment (smark88) [reply ops/k8s/deployments/grafana.yaml:75]:
```        
- name: GF_AUTH_GOOGLE_ALLOWED_DOMAINS
          valueFrom:
            secretKeyRef:
              name: googleClientidSecret
              key: allowedDomains
```

these are the specified domains

Comment (smark-broad) [reply ops/k8s/deployments/grafana.yaml]: You cannot camelCase the keys in the yaml

File: ops/k8s/configs/oauth.yaml.ctmpl (hunk 1 above)
Comment (smark88) [reply ops/k8s/configs/oauth.yaml.ctmpl:11]: You cannot camelCase the keys in the kube yamls

**Pull request merged**